src/js/containers/LoginPage/LoginPage.js

```diff
@@ -17,19 +37,126 @@ class LoginPage extends Component {
         }
     }
 
+    openLinkInWindow(link) {
+        window.open(link, '_blank', 'toolbar=0,status=0,width=720,height=700');
+    }
+
+    handleEmailChange(email) {
+        this.setState({ email });
+    }
+
+    handleApiKeyChange(apiKey) {
+        this.setState({ apiKey });
+    }
+
+    handleLoginSubmit(e) {
+        e.preventDefault();
+        const { dispatch, config } = this.props;
+
+        let isHostAPILogin = getConfigValue(config, 'useHostAPILogin');
```

Comment:
It doesn't matter much but we used `const` and `let` randomly in most cases. Just wanted point that out. Maybe _(probably not)_ in the feature it can be refactored.